firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -66,6 +67,7 @@
   private final IidStore iidStore;
   private final RandomFidGenerator fidGenerator;
   private final Object lock = new Object();
+  private final ExecutorService networkExecutor;
```

Comment:
Please cluster both the ExecutorServices below each other.

Reply:
Done